brave/brave-core
# Storybook: Created side-nav-component

Resolves <https://github.com/brave/brave-browser/issues/15023>

Created side nav component and added it too storybook.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that the storybook shows new component.
3. Check functionality of the new component.

https://user-images.githubusercontent.com/40611140/113228054-43a7aa80-9251-11eb-9db1-e29f46200771.mov
